server/chat-commands/moderation.ts

```diff
@@ -86,7 +81,7 @@ export const commands: ChatCommands = {
 			return this.errorReply(`User '${name}' is unregistered, and so can't be promoted.`);
 		}
 
-		const currentGroup = room.auth[userid] || Config.groupsranking[0];
+		const currentGroup = room.auth.get(userid) || Config.groupsranking[0];
```

Comment:
Should probably support `room.auth.get(userid, true)` to fall back to the default group.